vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -30,6 +31,18 @@ def get_services(inventory)
 
     def get_pods(inventory)
       process_collection(inventory["pod"], :container_groups) { |n| parse_pod(n) }
+      @data[:container_groups].each do |x|
```

Comment:
I'm not a big fan of using `x` as the iterator variable.  Would prefer something more contextual, like `cg` in this case.  Same goes for endpoints below.